### PR TITLE
Removed cluster dependencies

### PR DESCRIPTION
**Pull request by inthirakumaaran**
removed cluster cnf
removed cluster pom.xml and dependencies in the main pom.xml
removed the cluster readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,9 @@ You can run IS Performance Tests from the source using the following instruction
 ### Prerequisites
 
 * [Maven 3.5.0 or later](https://maven.apache.org/download.cgi)
-* [AWS CLI](https://aws.amazon.com/cli/) - Please make sure to [configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) and set the output format to `text`.
-* [Apache JMeter 4.0](https://jmeter.apache.org/) Setup tarball.
+* [AWS CLI](https://aws.amazon.com/cli/) - Please make sure to [configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) and set the output format to `json`.
+* [Apache JMeter 3.3](https://jmeter.apache.org/) Setup tarball.
+* wso2 is server zip file
```
Comment (vihanga-liyanage) [inline README.md]: WSO2 IS server zip file.

File: README.md [hunk 2]
```diff
@@ -31,16 +32,19 @@ You can run IS Performance Tests from the source using the following instruction
...
 cd performance-is
+git checkout single-node-performance
+```
+3. Build the artifacts using Maven.
+```
 mvn clean install
 ```
 
-3. Change directory to `cloudformation/` and run the `init-performance-tests.sh` script. Following is the basic command.
+4. Change directory to `cloudformation/` and run the `init-performance-tests.sh` script. Following is the basic command.
```
Comment (vihanga-liyanage) [inline README.md]: init-performance-tests.sh?

File: README.md [hunk 3]
```diff
@@ -49,7 +53,7 @@ See usage:
 ./init-performance-tests.sh -k <key_file> 
    -a <aws_access_key> -s <aws_access_secret>
    -c <certificate_name> -j <jmeter_setup_path>
-   [-n <wum_username>] [-e <wum_password>]
+   [-n <is_setup_path>]
```
Comment (vihanga-liyanage) [inline README.md]: [-n <IS_zip_file_path>]

File: README.md [hunk 4]
```diff
@@ -76,11 +79,11 @@ See usage:
 2. Run the CloudFormation template and creat the deployment, wait till the stack creation completes.
 3. Extract the following using the AWS CLI.
    * Bastion node public IP. (Used as the JMeter client)
-   * Load balancer hostname.
-   * Private IPs of WSO2 IS instances.
+   * Private IP of WSO2 IS instance.
```
Comment (vihanga-liyanage) [inline README.md]: Private IP of the WSO2 IS instance.

File: README.md [hunk 5]
```diff
@@ -76,11 +79,11 @@ See usage:
...
    * RDS instance hostname.
-4. Copy `setup-bastion.sh`, key file and the JMeter setup to bastion node.
-5. SSH into the bastion node and execute the copied [setup-bastion.sh](distribution/scripts/setup) script, which will setup the additional components in the deployment.
-6. SSH into the bastion node and execute the [run-performance-test.sh](distribution/scripts/jmeter) script, which will run the test and collect the results.
-7. Download the test results from the bastion node.
-8. Create the `summary.csv` file and the `summary.md` file. 
+4. Setup the wso2 IS server in the instance and create the databases
+5. Copy `setup-bastion.sh`, key file and the JMeter setup to bastion node.
```
Comment (vihanga-liyanage) [inline README.md]: I thought we are coping the whole target tar ball?

File: README.md (hunk 1 above)
Comment (inthirakumaaran) [reply README.md]: resolved

File: README.md (hunk 2 above)
Comment (inthirakumaaran) [reply README.md]: resolved

File: README.md (hunk 3 above)
Comment (inthirakumaaran) [reply README.md]: resolved

File: README.md (hunk 5 above)
Comment (inthirakumaaran) [reply README.md]: resolved

File: README.md (hunk 4 above)
Comment (inthirakumaaran) [reply README.md]: resolved

**Pull request merged**